Framework/Algorithms/src/Stitch1D.cpp

```diff
@@ -423,26 +420,14 @@ boost::tuple<int, int> Stitch1D::findStartEndIndexes(double startOverlap, double
  @return True if there are any non-zero errors in the workspace
  */
 bool Stitch1D::hasNonzeroErrors(MatrixWorkspace_sptr &ws) {
-  auto ws_size = static_cast<int64_t>(ws->getNumberHistograms());
-  bool hasNonZeroErrors = false;
-  PARALLEL_FOR_IF(Kernel::threadSafe(*ws))
-  for (int64_t i = 0; i < ws_size; ++i) {
-    PARALLEL_START_INTERRUPT_REGION
-    if (!hasNonZeroErrors) // Keep checking
-    {
-      const auto &e = ws->e(i);
-      auto it = std::find_if(e.begin(), e.end(), isNonzero);
-      if (it != e.end()) {
-        PARALLEL_CRITICAL(has_non_zero) {
-          hasNonZeroErrors = true; // Set flag. Should not need to check any more spectra.
-        }
-      }
+  for (auto i = 0u; i < ws->getNumberHistograms(); ++i) {
```

Comment:
I really like the detail of using unsigned int in the for loop, I'll start doing that myself from now on :)